### ISSUE TITLE
Feature: bridge isReady event or callback

### ISSUE THREAD

**Issue by codecomeon** [gronxb/webview-bridge#18]
A common scene is that:

When web page ready, app inject some data into web. Otherwise, app should prepare some async environment when web already done. For example lazy load modules in app, loading process will cost times.

Suggestion API Design:
- ready state getter, maybe current field 'isReady'
- ready state change event, maybe provide as subscription or event emitter.

**Comment by codecomeon** [comment]
Or app should show loading before bridge ready, and hide loading when bridge ready.

**Comment by gronxb** [comment]
Thank you!

https://gronxb.github.io/webview-bridge/using-a-native-method.html#checking-webview-bridge-availability

Something similar exists here, but it's not reactive.
It sounds like what you want is to observe the ready state either event-driven or state-driven so that it can work reactively.

I'll try to prepare that suggestion as well.

**Comment by gronxb** [comment]
Releaesd in [v1.2.0](https://www.npmjs.com/package/@webview-bridge/web).

## Simle Usage
```tsx
const nativeMethod = linkNativeMethod<AppBridge>({
  onReady: async (method) => {
    console.log("nativeMethod is ready");
    const version = await method.getBridgeVersion();
    console.log("currentBridgerVersion", version);
  },
});
```

However, integration with the state of each framework is not yet ready. Since it will be further integrated into the shared state, the issue will not be closed.

**Comment by gronxb** [comment]
Shared state was released in v1.3.1.
So you can use states like isReady to hide the UI. 🥰

https://gronxb.github.io/webview-bridge/shared-state/react-native.html
